src/interfaces/IFunnelFactory.sol

```diff
@@ -4,15 +4,6 @@ pragma solidity 0.8.17;
 /// @title Interface for Funnel Factory
 /// @author Zac (zlace0x), zhongfu (zhongfu), Edison (edison0xyz)
 interface IFunnelFactory {
-    /// Error thrown when funnel is not deployed
```

Comment:
Would prefer FunnelFactor errors to be here. 

IFunnel: Holds all Funnel errors
IFunnelFactory: Holds all FunnelFactory errors
FunnelErrors: Hold all shared/external errors